sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Features Added
 
+- Added support for more `AZURE_TOKEN_CREDENTIALS` environment variable values to specify a single credential type to use in `DefaultAzureCredential`: in addition to 'dev' and 'prod', possible values now include 'Environment', 'WorkloadIdentity', 'ManagedIdentity', and 'AzureCli' - each for the corresponding `*Credential` class.
```

Comment:
For consistency with the other languages, require the "Credential" suffix in the env var value. See https://github.com/Azure/azure-sdk-for-go/pull/24812 as an example.